cpp/doxygen/Doxyfile

```diff
@@ -0,0 +1,2427 @@
+# Doxyfile 1.8.11
```

Comment:
Didn't @jrhemstad 's PR #823 already add this file?

Reply:
yes it has his changes